mod/wikirate/lib/javascript/script_collapse.js.coffee

```diff
@@ -88,16 +88,16 @@ findCollapseTarget = (toggle) ->
     else
       $toggle.parent()
 
-  target = $toggle.data('target') || '.collapse'
+  if $toggle.attr("href") != null
+    return $($toggle.attr("href"))
+  target = $toggle.attr("href") || $toggle.data('target') || '.collapse'
   if $toggle.data("collapse")?
     target += $toggle.data("collapse")
-
-  if $toggle.find(target).length
+  else if $toggle.find(target).length
```

Comment:
maybe we could do this
 `href_attr = $toggle.attr("href")
  if typeof href_attr != typeof undefined && href_attr != false
    return $($toggle.attr("href"))`